website/docs/deployment.mdx

```diff
@@ -90,41 +90,41 @@ export default function Home() {
+- 🔴 Self-hosting requires experience in networking as well as Linux and web server administration. It's the most difficult option, and would require the most time to manage successfully. Expense-wise, cloud services are almost never free, and purchasing/deploying an onsite server can be even more costly.
+- 🟢 Jamstack providers can help you set up a working website in almost no time and offers features like server-side redirects that are easily configurable. Many providers offer generous build-time quotas even for free plans that you would almost never exceed. However, free plans have limits, and you would need to pay once you hit those limits. Check the pricing page of your provider for details.
+- 🟡 The GitHub Pages deployment workflow can be tedious to set up (Evidence: see the length of [Deploying to GitHub Pages](#deploying-to-github-pages)!). However, this service (including build and deployment) is always free for public repositories, and we have detailed instructions to help you make it work.
 
 </details>
 
 <details>
 
-<summary>How much server-side configuration would I need?</summary>
+<summary>How much server-side customization do I need?</summary>
 
-- 🟢 With self-hosting, you have access to the entire server's configuration. You can configure the virtual host to serve different content based on the request URL; you can do complicated server-side redirects; you can put part of the site behind authentication... If you need a lot of server-side features, self-host your website.
-- 🟡 Jamstack usually offers some server-side configurations, e.g. URLs formatting (trailing slashes), server-side redirects...
-- 🔴 GitHub Pages doesn't expose server-side configurations besides enforcing HTTPS and setting CNAME.
+- 🟢 With self-hosting, you have access to the entire server's configuration. You can configure the virtual host to serve different content based on the request URL, you can do complicated server-side redirects, you can implement authentication, and so on. If you need a lot of server-side features, self-host your website.
+- 🟡 Jamstack usually offers some server-side configuration (e.g. URL formatting (trailing slashes), server-side redirects, etc.).
+- 🔴 GitHub Pages doesn't expose server-side configuration besides enforcing HTTPS and setting CNAME records.
 
 </details>
 
 <details>
 
-<summary>Do I have needs to cooperate?</summary>
+<summary>Do I need collaboration-friendly deployment workflows?</summary>
 
-- 🟡 Self-hosted services can achieve the same effect as Netlify, but with much more heavy-lifting. Usually, you would have a specific person who looks after the deployment, and the workflow won't be very git-based as opposed to the other two options.
+- 🟡 Self-hosted services can leverage continous deployment functionality like Netlify, but more heavy-lifting is involved. Usually, you would designate a specific person to manage the deployment, and the workflow wouldn't be very git-based as opposed to the other two options.
```

Comment:
```suggestion
- 🟡 Self-hosted services can leverage continuous deployment functionality like Netlify, but more heavy-lifting is involved. Usually, you would designate a specific person to manage the deployment, and the workflow wouldn't be very git-based as opposed to the other two options.
```

In the original wording, we are trying to suggest that you can "self-implement Netlify" because you are really not using any CI/CD. I don't have strong opinions if you want to suggest incorporating Netlify, but just for you to consider

Reply:
Maybe this would be more accurate:

You can achieve similar functionality as Netlify with self-hosting services, but more heavy-lifting is involved . . . . [text continues]

I can't believe I misspelled "continuous." That's a little embarrassing.  :-O